Tests/Functional/Controller/ApiControllerTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace JMS\TranslationBundle\Tests\Functional\Controller;
+
+use JMS\TranslationBundle\Tests\Functional\BaseTestCase;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class ApiControllerTest extends BaseTestCase
+{
+    public function testUpdateAction()
+    {
+        // Add a file
+        $file = __DIR__.'/../Fixture/TestBundle/Resources/translations/navigation.en.yml';
+        file_put_contents($file, 'main.home: Home');
+
```

Comment:
Yes. Because we test if the file is being updated. So if you run the tests and then commit your changes you will accidentally commit the changed fixture file.